pennylane_lightning/lightning_tensor/_measurements.py

```diff
@@ -83,6 +83,28 @@ def expval(self, measurementprocess: MeasurementProcess):
         )._ob(measurementprocess.obs)
         return self._measurement_lightning.expval(ob_serialized)
 
+    def var(self, measurementprocess: MeasurementProcess):
+        """Variance of the supplied observable contained in the MeasurementProcess.
```

Comment:
Would you be able to include the limitations of this method for obs with only single-wire, projector and sparse-hamiltonian to the description?